.storybook/storybook-custom-theme.js

```diff
@@ -30,8 +32,7 @@ export default create({
   // inputBorder: 'white',
   // inputTextColor: 'white',
   // inputBorderRadius: 4,
-
   brandTitle: 'Pix',
-  brandUrl: 'https://pix.fr/',
-  brandImage: 'https://pix.fr/images/pix-logo.svg',
-});
\ No newline at end of file
+  brandUrl: 'https://pix.org',
```

Comment:
Pour éviter de sortir de Storybook :

```suggestion
  brandUrl: 'https://ui.pix.fr/',
```

Reply:
C'est ça qu'on veut ?

Reply:
Autant ne rien mettre du coup.

Reply:
En tout cas c'est ce qu'on a aujourd'hui (par erreur) mais ça me parait adapté

Reply:
J'ai rien mis du coup. Ca permet d'avoir le bon lien sur les RAs.